librz/bin/format/mach0/mach0.c

```diff
@@ -2443,11 +2443,11 @@ struct section_t *MACH0_(get_sections)(struct MACH0_(obj_t) * bin) {
 	if (!bin->sects) {
 		return NULL;
 	}
-	to = RZ_MIN(bin->nsects, 128); // limit number of sections here to avoid fuzzed bins
+	ut32 to = RZ_MIN(bin->nsects, 128); // limit number of sections here to avoid fuzzed bins
```

Comment:
What if `bin->nsects` is negative? Should we also check for this situation?

Reply:
Yes I think so 6f526ebdc24ccec8d1ebfa1ab1964e2f98b71778